lib/ansible/config/base.yml

```diff
@@ -175,7 +175,8 @@ CACHE_PLUGIN_TIMEOUT:
   type: integer
   yaml: {key: facts.cache.timeout}
 COLLECTIONS_SCAN_SYS_PATH:
-  name: enable/disable scanning sys.path for installed collections
+  name: Toggle to scan sys.path for installed collections
```

Comment:
name should be pretty version of constant
name: Path for python installed collections